src/CoreWCF.Primitives/src/CoreWCF/Telemetry/WcfInstrumentationEventSource.cs

```diff
@@ -0,0 +1,60 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+
+using System;
+using System.Diagnostics.Tracing;
+
+namespace CoreWCF.Telemetry;
+
+
+[EventSource(Name = "OpenTelemetry-Instrumentation-CoreWCF")]
```

Comment:
This is a bit of a philosophical question:- should these be ETW events which may be monitored on Windows, but will probably never be seen in a linux/container environment? The alternative would be to use ILogger and have them collected that way.

Reply:
I can easily remove them. I would not need those. I would just remove it, when it's ok. Those events are just for Filtering and Enrichment.